### DISCUSSION TITLE
GSoC 2024 Optimize Convex Hull Algorithm

### DISCUSSION THREAD

**Discussion by creataur** [elalish/manifold#770] [General]
Hello, my name is Pankaj Sharma and I am interested in contributing to this project. I have good foundation knowledge of algorithms and am eager to learn new material.
I would like to discuss the project and my application with potential mentors @zalo and @pca006132.
Thank you for your consideration.

**Comment by pca006132** [comment]
you can discuss in https://github.com/opencax/GSoC/issues/89
